<ui version="4.0" >
 <class>ColorYuvDialog</class>
 <widget class="QDialog" name="ColorYuvDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Avisynth ColorYUV</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Colour Correction Matrix</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="3" column="2" >
         <widget class="QSpinBox" name="brightnessVSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>-10000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Contrast</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" >
         <widget class="QSpinBox" name="gammaUSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>-10000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QSpinBox" name="brightnessYSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>-10000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QSpinBox" name="contrastYSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>-10000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Chroma U:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QSpinBox" name="gammaYSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>-10000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QSpinBox" name="contrastVSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>-10000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Chroma V:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4" >
         <widget class="QSpinBox" name="gainVSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>-10000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QSpinBox" name="contrastUSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>-10000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="3" >
         <widget class="QSpinBox" name="gammaVSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>-10000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="4" >
         <widget class="QSpinBox" name="gainUSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>-10000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="4" >
         <widget class="QSpinBox" name="gainYSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>-10000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Luma Y:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Brightness</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QSpinBox" name="brightnessUSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>-10000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="4" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Gain</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Gamma</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="5" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="5" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="5" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Note: all values are normalised to 256</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Parameters</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Matrix:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QComboBox" name="levelComboBox" >
          <item>
           <property name="text" >
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>TV -> PC</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>PC -> TV</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>PC -> TV.Y</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_10" >
          <property name="text" >
           <string>Level:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="matrixComboBox" >
          <item>
           <property name="text" >
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Rec.709</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="optComboBox" >
          <item>
           <property name="text" >
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Coring</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Opt:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="autoGainCheckBox" >
        <property name="text" >
         <string>Auto Gain</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="colourStatsCheckBox" >
        <property name="text" >
         <string>Display Colour Statistics</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="centreOffsetsCheckBox" >
        <property name="text" >
         <string>Centre Colour Offsets</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>contrastYSpinBox</tabstop>
  <tabstop>brightnessYSpinBox</tabstop>
  <tabstop>gammaYSpinBox</tabstop>
  <tabstop>gainYSpinBox</tabstop>
  <tabstop>contrastUSpinBox</tabstop>
  <tabstop>brightnessUSpinBox</tabstop>
  <tabstop>gammaUSpinBox</tabstop>
  <tabstop>gainUSpinBox</tabstop>
  <tabstop>contrastVSpinBox</tabstop>
  <tabstop>brightnessVSpinBox</tabstop>
  <tabstop>gammaVSpinBox</tabstop>
  <tabstop>gainVSpinBox</tabstop>
  <tabstop>matrixComboBox</tabstop>
  <tabstop>optComboBox</tabstop>
  <tabstop>levelComboBox</tabstop>
  <tabstop>autoGainCheckBox</tabstop>
  <tabstop>colourStatsCheckBox</tabstop>
  <tabstop>centreOffsetsCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColorYuvDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColorYuvDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
